### ISSUE TITLE
RocketAPI.csproj : error

### ISSUE THREAD

**Issue by li81** [NecronomiconCoding/NecroBot#646]
NecroBot-master\FeroxRev\PokemonGo.RocketAPI\PokemonGo.RocketAPI.csproj : error  : The project file could not be loaded. Could not find a part of the path.

Im using Visual Studio 2015. Anyone can help me out? I'm extracting the files and open the project, then this error shows up.

**Comment by padmanek** [comment]
@li81 It's all in the wiki. Read https://github.com/NecronomiconCoding/NecroBot/wiki/Getting-Started#getting-started and close the issue.

**Comment by li81** [comment]
That's exactly what I have followed...

**Comment by mrsean77** [comment]
As padmanek said, look in the Wiki installation section:

1. Get the current source code from Code.
2. Get the current version of the RocketAPI we use from RocketAPI
3. Unzip the source code for NecroBot
4. Unzip the RocketAPI so the folder structure matches NecroBot\FeroxRev\PokemonGo.RocketAPI\

TL;DR get the RocketAPI yourself and follow the instructions above
https://github.com/FeroxRev/Pokemon-Go-Rocket-API/archive/master.zip

**Comment by padmanek** [comment]
@li81 Then you've not followed close enough. You've most likely put PokemonGo.RocketAPI in the wrong folder. BTW. Why dont you use already compiled version? It doesn't need messing around with Visual Studio if you're not fluent with it. https://github.com/NecronomiconCoding/NecroBot/releases

**Comment by Loham** [comment]
Unless you're trying to tinker with source code to improve the engine or troubleshoot something just use the release version.

**Comment by li81** [comment]
It's all sorted, thanks for the support.
